Controller/DefaultController.php

```diff
@@ -99,13 +100,13 @@ protected function getResque()
      *
      * @return array
      */
-    private function getShowParameters()
+    private function getShowParameters(Request $request)
```

Comment:
Does anyone know if this injection is valid for the supported Symfony versions? >2.0 I can't seem to find when it was first introduced.